frontend-react/e2e/pages/authenticated/admin/organization-receiver-message-test.ts

```diff
@@ -87,8 +95,13 @@ export class OrganizationReceiverMessageTestPage extends BasePage {
         ];
     }
 
-    createMockTestSubmissionHandler(isFailed = false): RouteHandlerFulfillEntry {
-        const result = isFailed ? errorMessageResult : passMessageResult;
+    createMockTestSubmissionHandler(resultType: "pass" | "fail" | "warn" = "pass"): RouteHandlerFulfillEntry {
```

Comment:
nit: I'm not a huge fan of nested ternaries since they're a little hard to follow, maybe a `switch` is cleaner:

```
switch (resultType) {
    case "fail":
      result = errorMessageResult;
      break;
    case "warn":
      result = warningMessageResult;
      break;
    default:
      result = passMessageResult;
      break;
  }
```